assets/app/scripts/filters/resources.js

```diff
@@ -108,6 +124,12 @@ angular.module('openshiftConsole')
       }
     };
   })
+  .filter('imageIconClass', function(imageStreamTagAnnotationFilter) {
```

Comment:
`imageStreamTagIconClass`... sorry for the verbosity, but `image` may be the most overloaded term right now... don't want any confusion over what the filter does